compass/landice/mesh.py

```diff
@@ -304,8 +306,8 @@ def get_dist_to_edge_and_GL(self, thk, topg, x, y, window_size=1.e5):
         irng = irng[np.nonzero(np.logical_and(irng >= 0, irng < ny))]
         jrng = jrng[np.nonzero(np.logical_and(jrng >= 0, jrng < nx))]
 
-        dist_to_here = ((XPOS[np.ix_(irng, jrng)] - x[j])**2 +
-                        (YPOS[np.ix_(irng, jrng)] - y[i])**2)**0.5
+        dist_to_here = ((XPOS[np.ix_(irng, jrng)] - x[j]) ** 2 +
+                        (YPOS[np.ix_(irng, jrng)] - y[i]) ** 2) ** 0.5
```

Comment:
Out of curiosity, is this one that PEP8 complained about?  I would never put spaces around a `**` operator myself.  I don't think it's very intuitive.  So it would be annoying to me to find out that I'm breaking the rules.

Reply:
@xylar  PEP8 didn't complain about this. There is another line that puts spaces around `**` (`speed = (vx ** 2 + vy ** 2) ** 0.5`) and multiple lines that put spaces round `* and other operaters `-`, `/`, etc. I can change them if you prefer. And if we decide to change them, we would also need to change the same `mesh.py` files in the Greenland-mesh-generation test case.

Reply:
Feel free to leave it. I agree, the important thing is to be consistent. I like spaces around the other operators.